src/components/10-atoms/icon/story.js

```diff
@@ -28,7 +28,14 @@ export default {
 
 export const Icon = () => {
   const showAll = boolean('Show list of available icons', false);
-  const noIcon = boolean('Do not set icon on component', false);
+  const noIcon = boolean('Do not set attribute "icon"', false);
+  const noSize = boolean('Do not set attribute "size"', false);
```

Comment:
What are those for? To explicitly remove the dom attributes?

Reply:
Yes. Needed to test the default behaviour.